dnf/module/exceptions.py

```diff
@@ -0,0 +1,92 @@
+# Public License for more details.  You should have received a copy of the
+# GNU General Public License along with this program; if not, write to the
+# Free Software Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA
+# 02110-1301, USA.  Any Red Hat trademarks that are incorporated in the
+# source code or documentation are not subject to the GNU General Public
+# License and may only be used or replicated with the express permission of
+# Red Hat, Inc.
+#
+
+import dnf
+
+
+class LoadCacheException(dnf.exceptions.Error):
+    def __init__(self, cache_dir):
+        value = "Cannot load from cache dir: {}".format(cache_dir)
+        super(LoadCacheException, self).__init__(value)
+
+
+class MissingYamlException(dnf.exceptions.Error):
+    def __init__(self, cache_dir):
+        value = "Missing file *modules.yaml in metadata cache dir: {}".format(cache_dir)
+        super(MissingYamlException, self).__init__(value)
+
+
+class NoModuleException(dnf.exceptions.Error):
+    def __init__(self, module_spec):
+        value = "No such module: {}".format(module_spec)
+        super(NoModuleException, self).__init__(value)
+
+
+class NoStreamException(dnf.exceptions.Error):
+    def __init__(self, stream):
+        value = "No such stream: {}".format(stream)
+        super(NoStreamException, self).__init__(value)
+
+
+class EnabledStreamException(dnf.exceptions.Error):
+    def __init__(self, module_spec):
+        value = "No enabled stream for module: {}".format(module_spec)
+        super(EnabledStreamException, self).__init__(value)
+
+
+class VersionLockedException(dnf.exceptions.Error):
+    def __init__(self, module_spec, version):
+        value = "'{}' is locked to version: {}".format(module_spec, version)
+        super(VersionLockedException, self).__init__(value)
+
+
+class NoProfileException(dnf.exceptions.Error):
+    def __init__(self, profile):
+        value = "No such profile: {}".format(profile)
+        super(NoProfileException, self).__init__(value)
+
+
+class ProfileNotInstalledException(dnf.exceptions.Error):
+    def __init__(self, module_spec):
+        value = "Profile not installed: {}".format(module_spec)
+        super(ProfileNotInstalledException, self).__init__(value)
+
+
+class NoStreamSpecifiedException(dnf.exceptions.Error):
+    def __init__(self, module_spec):
+        value = "No stream specified for '{}', please specify stream".format(module_spec)
+        super(NoStreamSpecifiedException, self).__init__(value)
+
+
+class NoProfileSpecifiedException(dnf.exceptions.Error):
+    def __init__(self, module_spec):
+        value = "No profile specified for '{}', please specify profile".format(module_spec)
+        super(NoProfileSpecifiedException, self).__init__(value)
+
+
+class PossibleProfilesExceptions(dnf.exceptions.Error):
+    def __init__(self, module_spec, profiles):
+        value = "No such profile: {}. Possible profiles: {}".format(module_spec, profiles)
```

Comment:
There were a request from translation team to do something like. And there is no set translation of that strings. 
```
msg = _("My name is {name} and I am {what_i_am}")
msg.format(name="HaHa", what_i_am="nobady")
```

Reply:
I've been talking to one of translators and it is something they can do on their own without requiring us to do anything, so irrelevant

Reply:
let's keep it as it is